### ISSUE TITLE
operator ignoring CCPImagePrefix

### ISSUE THREAD

**Issue by k1ng440** [CrunchyData/postgres-operator#270]
CCPImagePrefix is being ignored in manual deployment and downloading from crunchydata/crunchy-postgres:centos7-10.4-1.8.3

```
kd secrets apiserver-conf-secret -n pgo
Name:         apiserver-conf-secret
Namespace:    pgo
Labels:       <none>
Annotations:  <none>

Type:  Opaque

Data
====
pgo.load-template.json:   2072 bytes
pgo.lspvc-template.json:  987 bytes
pgo.yaml:                 1198 bytes
pgorole:                  472 bytes
pgouser:                  31 bytes
server.crt:               1334 bytes
server.key:               1704 bytes
```

```
Cluster:
  CCPImagePrefix:  cryptovalue
  CCPImageTag:  centos7-10.4-1.8.3
  Port:  5432
  User:  cryptovalue
  Database:  cryptovalue
  PasswordLength:  8
  Strategy:  1
  Replicas:  0
  ArchiveMode:  false
  ArchiveTimeout:  60
PrimaryStorage: storage1
BackupStorage: storage1
ReplicaStorage: storage1
Storage:
  storage1:
    StorageClass: fast
    AccessMode:  ReadWriteOnce
    Size:  100Gi
    StorageType:  create
    Fsgroup: 26
    SupplementalGroups:  65534
  storage2:
    AccessMode:  ReadWriteOnce
    Size:  333M
    StorageType:  dynamic
    StorageClass:  gluster-heketi
    Fsgroup:  26
  storage3:
    AccessMode:  ReadWriteOnce
    Size:  440M
    StorageType:  dynamic
    StorageClass:  fast
    Fsgroup:  26
DefaultContainerResource: 
ContainerResources:
  small:
    RequestsMemory:  512Mi
    RequestsCPU:  0.1
    LimitsMemory:  512Mi
    LimitsCPU:  0.1
  large:
    RequestsMemory:  2Gi
    RequestsCPU:  2.0
    LimitsMemory:  6Gi
    LimitsCPU:  4.0
Pgo:
  AutofailSleepSeconds:  9
  Audit:  true
  Metrics:  true
  LSPVCTemplate:  /config/pgo.lspvc-template.json
  LoadTemplate:  /config/pgo.load-template.json
  COImagePrefix:  crunchydata
  COImageTag:  centos7-3.1
```

**Comment by k1ng440** [comment]
StorageClass also being ignored.

**Comment by jmccormick2001** [comment]
what is happening is I forgot to make the operator container use the settings within the pgo.yaml file for the CCPImagePrefix and COImagePrefix and COImageTag settings...instead its pulling those from the env vars specified in the deployment.yaml file.  I will have this fixed in 3.2 where both the apiserver and operator containers will both look for those values ONLY in the pgo.yaml config file.  Till that release, update the deployment.yaml env vars in the operator container to match the values in pgo.yaml and it should work.  I just merged in code into pre32 branch to fix this one.

**Comment by k1ng440** [comment]
ok. thanks
